pkg/syncer/specsyncer.go

```diff
@@ -151,27 +151,24 @@ func (c *Controller) applyToDownstream(ctx context.Context, gvr schema.GroupVers
 
 	downstreamObj := upstreamObj.DeepCopy()
 	downstreamObj.SetUID("")
+	downstreamObj.SetCreationTimestamp(metav1.Time{})
 	downstreamObj.SetResourceVersion("")
 	downstreamObj.SetNamespace(downstreamNamespace)
 	downstreamObj.SetManagedFields(nil)
 	downstreamObj.SetClusterName("")
-
-	ownedByLabel := downstreamObj.GetLabels()["kcp.dev/owned-by"]
-	var ownerReferences []metav1.OwnerReference
-	for _, reference := range downstreamObj.GetOwnerReferences() {
-		if reference.Name == ownedByLabel {
-			continue
-		}
-		ownerReferences = append(ownerReferences, reference)
-	}
-	downstreamObj.SetOwnerReferences(ownerReferences)
+	// Deletion fields are immutable and set by the downstream API server
+	downstreamObj.SetDeletionTimestamp(nil)
```

Comment:
If they are immutable, do they need to be cleared?

Reply:
I think so. The update requests get rejected by the downstream API servers otherwise.

Reply:
By removing them, they get removed from the SSA patch, do that there are left untouched downstream.